x-pack/plugins/security_solution/server/lib/detection_engine/signals/search_after_bulk_create.ts

```diff
@@ -186,6 +185,17 @@ export const searchAfterAndBulkCreate = async ({
           break;
         }
 
+        // make sure we are not going to create more signals than maxSignals allows
+        if (
+          signalsCreatedCount != null &&
```

Comment:
`signalsCreatedCount` can never be `null`/`undefined` here, no?

Reply:
Yes that is correct. I'll update this thanks!